python/cuml/dask/common/base.py

```diff
@@ -283,10 +284,14 @@ def _run_parallel_func(self,
         #  `to_delayed_output()` function
         # TODO: Add eager path back in
 
+        if output_collection_type == 'series':
+            # This returns cudf.Series as with LabelEncoder
+            preds = [pred.values for pred in preds]
+            output_collection_type = 'cupy'
```

Comment:
Sorry, maybe I'm misunderstanding output_collection_type -- we should get a cudf series out in the end, right?

Reply:
I think we need to resolve this before finalizing a review. It's important to be able to deliver a Series output when request, as that's what the single-gpu variants do.